python/pyspark/mllib/linalg.py

```diff
@@ -208,9 +209,55 @@ def __init__(self, ar):
             ar = ar.astype(np.float64)
         self.array = ar
 
+    def toString(self):
+        """
+        Convert DenseVector to string representation.
+
+        >>> a = DenseVector([0, 1, 2, 3])
+        >>> a.toString()
+        '[0.0,1.0,2.0,3.0]'
+        """
+        return str(self)
+
+    def copy(self):
```

Comment:
I would guess we don't need this either.  But I'll ping @mengxr --- How does this differ from ```copy.deepcopy()```?